.github/workflows/ci.yml

```diff
@@ -21,12 +21,12 @@ jobs:
           cache: 'npm'
       - run: npm ci || npm install
       - run: npm test
-      - run: npm run lint
       - run: npm run build
       - name: Check bundle freshness
         run: |
-          if ! git diff --quiet -- resources/modules/ext.LabkiPackManager/app.bundle.js resources/css/labkipackmanager.css resources/css/lpm-sfc.css; then
+          if ! git diff --quiet -- resources/modules/ext.LabkiPackManager/app.bundle.js resources/css/labkipackmanager.css; then
             echo "::warning ::Built assets are out of date. Please run 'npm run build' and commit the changes."
+            exit 1
```

Comment:
previously this would not actually fail in a visible way when the compiled assets needed to be rebuilt.